docs/index.rst

```diff
@@ -0,0 +1,497 @@
+CUB
+==================================================
+
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+
+   ${repo_docs_api_path}/CUB_api
+   developer_overview
+   test_overview
+
+.. the line below can be used to use the README.md file as the index page
+.. .. mdinclude:: ../README.md
+
+What is CUB?
+==================================================
+
+CUB provides state-of-the-art, reusable software components for every layer 
+of the CUDA programming model:
+
+* **Parallel primitives**
+
+  * **Warp-wide "collective" primitives**
```

Comment:
Do Doxygen groups no longer work with the Sphinx infrastructure? 

I'm comparing the new docs to the old docs page, and these used to link to the Warp/Block/Device modules from Doxygen, e.g., https://nvlabs.github.io/cub/group___warp_module.html

Reply:
I haven't found how to setup automatic grouping yet.